### PR TITLE
Game clock

### PR DESCRIPTION
**Pull request by mfountoulakis**

### CODE REVIEW

File: pages/game.js [hunk 1]
```diff
@@ -23,22 +31,52 @@ class Game extends React.Component {
...
       game,
       game: {
-        basicGameData: { hTeam, vTeam, clock },
+        basicGameData: { hTeam, vTeam },
       },
     } = this.props;
 
-    return (
+    const { clock, period } = this.state;
+    return period ? (
```
Comment (jssee) [inline pages/game.js]: I think maybe a null check would be cleaner and easier to read
```suggestion
    return ({ !!period && ... })
```

File: pages/game.js [hunk 2]
```diff
@@ -23,22 +31,52 @@ class Game extends React.Component {
...
+    const url = `${publicRuntimeConfig.BASEURL}`;
+
+    const result = await fetch(
+      `${url}/api/boxscore/${game.basicGameData.gameId}`,
+    );
+    const g = await result.json();
+    return g;
+  };
+
+  componentDidMount() {
```
Comment (jssee) [inline pages/game.js:51]: this is nice, but my instincts tell me this logic should happen in `Scoreboard` component. Its already taking in the `game` data that this is operating on and handling the display. What do you think?

Comment (mfountoulakis) [reply pages/game.js:51]: My plan for this is in here:
https://github.com/mfountoulakis/hoopify/blob/gameText/pages/game.js
on line 77: 

We could poll for the endpoint every minute, once the game becomes active, we present the scoreboard, if not, we display:

```{`The ${teamNames[hTeam.triCode]} take on the ${
            teamNames[vTeam.triCode]
          } @ ${arena.name}. Game
            starts at ${startTimeEastern}`}```

We just don't want a situation where we'd have to poll the same endpoint twice. We could move the polling to the scoreboard if you want to move all the presentation there. How do you think it should look if we moved it?

File: pages/game.js (hunk 1 above)
Comment (mfountoulakis) [reply pages/game.js]: i like the null check!

File: pages/game.js (hunk 2 above)
Comment (jssee) [reply pages/game.js:51]: ah word, gotcha. that does makes sense, lets move forward with this route.

**Pull request merged**